portfolio/src/main/webapp/comment.js

```diff
@@ -1,3 +1,38 @@
+/**
+ * Gets the log in status from "/login".
+ */
+function getLoginStatus() {
+    fetch("/login", {headers: {"Content-Type": "application/json"}}).
+    then(response => response.json()).then((json) => {
+        let commentSection = document.getElementById("content");
+        // Hides the comment section.
+        commentSection.style.display = "none";
+
+        let isLoggedIn = json.isLoggedIn;
+        let htmlText = json.htmlText;
+
+        if (!isLoggedIn) {
```

Comment:
Maybe abstract these show UI code to a separate method? 
```
if (isLoggedIn) { 
  showMemberUI(username); 
} else { 
  showGuestUI();
}
```

Reply:
Fixed